controllers/suite_test.go

```diff
@@ -86,6 +86,9 @@ func createOperatorNamespace(ramenNamespace string) {
 }
 
 var _ = BeforeSuite(func() {
+	// default controller type to DRHub
+	ramencontrollers.ControllerType = ramendrv1alpha1.DRHub
+
```

Comment:
(nit) could set `ramencontrollers.ConfigMapName = ramencontrollers.HubOperatorConfigMapName` here to avoid if on ramenconfig.go line 266